TOMP-API.yaml

```diff
@@ -1651,38 +1734,37 @@ components:
         meta:
           type: array
           items:
-            $ref: '#/components/schemas/key-value'
+            $ref: '#/components/schemas/keyValue'
 
-    amount-of-money:
+    amountOfMoney:
       type: object
       properties:
         amount:
-          description: This should be in the base unit as defined by the ISO 4217 currency code with the appropriate number of decimal places and omitting the currency symbol. e.g. if the price is in US Dollars the price would be 9.95. This is inclusive VAT   
+          description: This should be in the base unit as defined by the ISO 4217 currency code with the appropriate number of decimal places and omitting the currency symbol. e.g. if the price is in US Dollars the price would be 9.95. This is inclusive VAT
           type: number
           example: 9.95
-        amount-ex-vat:
+        amountExVat:
           type: number
           example: 8.95
-        currency-code:
+        currencyCode:
           description: ISO 4217 currency code
           type: string
           minLength: 3
-          maxLength: 3 
-        vat-rate:
+          maxLength: 3
+        vatRate:
           type: number
           description: value added tax rate (percentage of amount)
           example: 21.0
-        vat-country-code:
+        vatCountryCode:
           $ref : '#/components/schemas/country'
 
-    key-value:
+    keyValue:
       type: object
-      properties:
-        key:
-          type: string
+      minProperties: 1
+      maxProperties: 2
```

Comment:
Why 2?

Reply:
It can be 1. Should we provide a seperate field for description?